commitizen/providers/scm_provider.py

```diff
@@ -24,14 +25,7 @@ class ScmProvider(VersionProvider):
     It is meant for `setuptools-scm` or any package manager `*-scm` provider.
     """
 
-    TAG_FORMAT_REGEXS = {
-        "$version": r"(?P<version>.+)",
-        "$major": r"(?P<major>\d+)",
-        "$minor": r"(?P<minor>\d+)",
-        "$patch": r"(?P<patch>\d+)",
-        "$prerelease": r"(?P<prerelease>\w+\d+)?",
-        "$devrelease": r"(?P<devrelease>\.dev\d+)?",
-    }
+    TAG_FORMAT_REGEXS = get_tag_regexes(r"(?P<version>.+)")
```

Comment:
What will this match? It lookw like this key does not exists in the dict in `get_tag_regexes`

Reply:
This is the regex being passed in to be used for $version or ${version} in the return dictionary of part names to regex mappings. In the refactor to bring in a single definition of the TAG_FORMAT_REGEXS I needed to accept this as an input due to in this one situation the regex for it here already being that value, I am unsure why we have that one here and couldn't work it out so just replicated the current behaviour.